src/workloads/query/OneMDocCollection_LargeDocIntId.yml

```diff
@@ -0,0 +1,126 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query"
+Description: |
+  This workload tests performance of IDHACK on queries with int _id. Since this workloads loads
+  very large documents (average 10KB), you may need to free up disk space before running this
+  workload locally.
+
+KeyWords:
+- IDHACK
+
+Clients:
+  Default:
+    QueryOptions:
+      maxPoolSize: 2000
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  MaxPhases: &MaxPhases 5
+
+ActorTemplates:
+- TemplateName: IDHACK
+  Config:
+    Name: {^Parameter: {Name: "Name", Default: "IDHackQuery"}}
+    Type: CrudActor
+    Threads: {^Parameter: {Name: "Threads", Default: 1}}
+    Phases:
+      OnlyActiveInPhases:
+        Active: [{^Parameter: {Name: "OnlyActiveInPhase", Default: {unused: "please specify in which phases this actor should be active."}}}]
+        NopInPhasesUpTo: *MaxPhases
+        PhaseConfig:
+          Repeat: 1000
```

Comment:
Oh man I keep finding new things I am sorry. 1000 queries is not much... Can we make this 30 seconds or 1 minute? It will probably help to stabilize the results - I hope.